### PR TITLE
updated bank statuses with payment product swedish-giro status

### PR DESCRIPTION
**Pull request by janeklof**

### CODE REVIEW

File: docs/banks/NDEASESS.md [hunk 1]
```diff
@@ -63,6 +63,7 @@ sidebar_label: Nordea
 | Payment Product | Sandbox | Production |
 |---------------------|---|---|
 |domestic              | ![](https://img.shields.io/badge/status-active-success.svg) | ![](https://img.shields.io/badge/status-active-success.svg) |
+|swedish-giro          | ![](https://img.shields.io/badge/status-active-success.svg) | ![](https://img.shields.io/badge/status-in_development-yellow.svg) |
```
Comment (henrikwidlund) [inline docs/banks/NDEASESS.md:66]: Do we have a flag that indicates that it is not supported by the ASPSP yet? Our implementation is actually done.

File: docs/banks/SWEDSESS.md [hunk 2]
```diff
@@ -62,6 +62,7 @@ sidebar_label: Swedbank
 | Payment Product | Sandbox | Production |
 |---------------------|---|---|
 |domestic              | ![](https://img.shields.io/badge/status-active-success.svg) | ![](https://img.shields.io/badge/status-active-success.svg) |
+|swedish-giro          | ![](https://img.shields.io/badge/status-active-success.svg) | ![](https://img.shields.io/badge/status-active-success.svg) |
```
Comment (henrikwidlund) [inline docs/banks/SWEDSESS.md:65]: Giro is broken at the ASPSP in production.

Comment (janeklof) [reply docs/banks/SWEDSESS.md]: Now reflected in status

File: docs/banks/NDEASESS.md (hunk 1 above)
Comment (janeklof) [reply docs/banks/NDEASESS.md]: Changed to "implemented:waiting for aspsp release"

**Pull request merged**